src/mm-dialog/mm-dialog.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function(scope) {
+
+	scope.Dialog = Polymer({
+		is: 'mm-dialog',
+
+		iconMap: {
+			info:    { type: "info",    color: Colors.D3 },
+			success: { type: "success", color: Colors.B6 },
+			warning: { type: "warning", color: Colors.E5 },
+			error:   { type: "warning", color: Colors.C3 }
+		},
+
+		properties: {
+			type: {
+				type: String,
+				value: ''
+			},
+			header: {
+				type: String,
+				value: ''
+			},
+			width: {
+				type: Number,
+				value: 450
+			},
+			hidden: Boolean,
+			dismiss: Boolean,
```

Comment:
So this should probably be false as the default - ie - hide the "X" unless the dev specifies